### PR TITLE
Labbawi krestos

### PR DESCRIPTION
**Pull request by MershaMengistie**
This is my first encoding experience. I encoded only the basic features and gladly want to have your comments.

### CODE REVIEW

File: new/PRS13313Labbawekrestos.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <revisionDesc>
+            <change who="MA" when="2020-12-18">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ለባዌ ክርስቶስ</persName>
+                    <persName xml:lang="gez" corresp="#n1" type="normalized">Labbāwe Krǝstos</persName>
```
Comment (DenisNosnitsin1970) [inline new/PRS13313Labbawekrestos.xml:49]: Does he have any relations?

File: new/PRS13314Haylamasqal.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ኃይለ መስቀል</persName>
+                    <persName xml:lang="gez" corresp="#n1" type="normalized">Ḫāyla Masqal</persName>
+                    <faith type="EOTC"/>
+                  </person>
+                <listRelation>
+                    <relation name="saws:isCopierOf" active="PRS13314Haylamasqal" passive="QM001"/>
```
Comment (DenisNosnitsin1970) [inline new/PRS13314Haylamasqal.xml:53]: What is QM?

File: new/PRS13313Labbawekrestos.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="MA" when="2020-12-18">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ለባዌ ክርስቶስ</persName>
```
Comment (DenisNosnitsin1970) [inline new/PRS13313Labbawekrestos.xml:48]: Here and below, I see irregularities in the word dividers, please check.

File: new/PRS13328AbalaMaryam.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        <revisionDesc>
+            <change who="MA" when="2021-01-12">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">አባለ ማርያም</persName>
+                    <persName xml:lang="gez" corresp="#n1" type="normalized">ʾAbāla MĀryĀm</persName>
```
Comment (DenisNosnitsin1970) [inline new/PRS13328AbalaMaryam.xml:49]: why capitals?

File: new/PRS13329Pestqis.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="MA" when="2021-01-12">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ጵስጥቂስ</persName>
```
Comment (DenisNosnitsin1970) [inline new/PRS13329Pestqis.xml:48]: No word divider after the name?

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/PRS13313Labbawekrestos.xml (hunk 1 above)
Comment (MershaMengistie) [reply new/PRS13313Labbawekrestos.xml:49]: The relations are encoded in the other persons records.

File: new/PRS13314Haylamasqal.xml (hunk 2 above)
Comment (MershaMengistie) [reply new/PRS13314Haylamasqal.xml:53]: There is a parallel request for this manuscript and QM stands for the monastery i.e. Qäranǝyo Mädhanealem in Gojjaṃ

**Comment by MershaMengistie** [comment]
@DenisNosnitsin1970: can my encoded ms be merged?

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**